confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -338,10 +343,10 @@ def get_version(self, subject, avro_schema):
 
         result, code = self._send_request(url, method='POST', body=body)
         if code == 404:
-            log.error("Not found:" + str(code))
+            log.error("Schema for subject '{}' not found. Error code: {}".format(subject, code))
```

Comment:
Do you think we should begin addressing that level of granularity, or stick with the generic `Not found`?

Reply:
I'm fine with stating either the schema or the subject were not found. If memory serves this is what we do further down in the code. 

My only issue with the current wording is it could leave the unsuspecting user to believe that the schema was not registered at all. This as you and I know is not entirely correct; most users however do not know, nor should they care, how the schemas are stored.